account_banking_pain_base/models/account_payment_order.py

```diff
@@ -122,6 +125,28 @@ def _prepare_field(self, field_name, field_value, eval_ctx,
             value = value[0:max_size]
         return value
 
+    @api.model
+    def except_messages_prepare_field(self, eval_ctx, field_name):
+        """
+        Inherit this method to provide more detailed error messages for
+        exceptions to be raised while evaluating `field_name` using `eval_ctx`.
+        :return: List containing the error messages.
+        """
+        error_messages = list()
+        line = eval_ctx.get('line')
+        if line:
+            error_messages.append(
+                _("Cannot compute the '%s' of the "
```

Comment:
This text is redundant across messages. I was thinking of joining all the messages (including the one you have put right now outside), but only adding the significant part (second line).